src/styles/blocks/page.scss

```diff
@@ -0,0 +1,73 @@
+.page {
+  font-family: "Montserrat", sans-serif;
+  scroll-behavior: smooth;
+
+  &__body {
+    margin: 0;
+
+    &--with-menu {
+      overflow: hidden;
+    }
+  }
+
+  &__menu {
+    position: absolute;
+    top: 0;
+    right: 0;
+    left: 0;
+    z-index: 999;
+    transform: translateX(-100%);
+    transition: transform 0.3s;
+
+    &:target {
+      transform: translateX(0);
+
+      &::after {
+        content: "";
+        position: fixed;
+        left: 0;
+        right: 0;
+        z-index: 0;
+        height: 100vh;
+        background-color: rgba(0, 0, 0, 0.5);
+      }
+    }
+  }
+
+  &__section {
+    margin-bottom: 100px;
+
+    @include onDesktop {
+      margin-bottom: 200px;
+    }
+
+    &--title {
+      font-family: "Montserrat", sans-serif;
+      font-style: normal;
+      font-weight: 700;
+      font-size: 36px;
+      line-height: 36px;
+      text-align: center;
+      color: #0f0e08;
+      margin: 0 0 40px;
+
+      @include onTablet {
```

Comment:
It seems that the display and justify-content properties are the same for both onTablet and onDesktop media queries. Consider merging them into one media query with the appropriate min-width value.